blocklist.csv

```diff
@@ -563,10 +630,11 @@
 125680703,SkodaFrance,ŠKODA France
 124176955,G5games,G5 Games
 123818293,DisneylandParis,Disneyland Paris
-123193555,EA_Benelux,EA Benelux
-123093566,datadoghq,|Datadog, Inc.|
+123193555,EA_Benelux,EA Benelux, a PTC Technology|
```

Comment:
This line seems a bit weird, looks like the merge displaced this bit from line 524